package/kernel/linux/modules/hwmon.mk

```diff
@@ -45,6 +45,7 @@ $(eval $(call KernelPackage,hwmon-ad7418))
 
 define KernelPackage/hwmon-adt7410
   TITLE:=ADT7410 monitoring support
+  DEPENDS:=+LINUX_6_1:kmod-regmap-core
   KCONFIG:= \
 	CONFIG_SENSORS_ADT7X10 \
 	CONFIG_SENSORS_ADT7410
```

Comment:
This did not function for me unless it was added to the `$(call AddDepends/hwmon` below